src/pages/projects.js

```diff
@@ -775,7 +775,34 @@ const ProjectDescriptions = () => (
         </div>
       </div>
     </div>
+
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">eCapture</TitleWithAnchor>
```

Comment:
I think this should be moved up in alphabetical order instead of appended at bottom (hmm, looks like bumblebee is wrong too in that respect)

Reply:
Fyi, Dave and I discussed that a better ordering around emerging should be based on GH stars (as in highest to the top) rather than just alphabetical, so I'll go ahead merge this one as-is, and then follow-up with a small PR to do the sorting.